assets/css/base/_typography.scss

```diff
@@ -34,65 +92,72 @@
   letter-spacing: 0.25px;
 }
 
-a, textarea {
+a,
+textarea {
   color: $default-text-color;
 }
 
 i {
-  vertical-align: middle
+  vertical-align: middle;
 }
 
-h1, .h1 {
+h1,
+.h1 {
   font-size: 2rem;
   font-weight: 200;
   letter-spacing: -1.5px;
 }
 
-h2, .h2 {
+h2,
+.h2 {
   font-size: 1.375rem;
   font-weight: normal;
   letter-spacing: -0.5px;
 }
 
-h3, .h3 {
+h3,
+.h3 {
   font-size: 1.25rem;
   font-weight: normal;
   letter-spacing: 0;
 }
 
-h4, .h4 {
+h4,
+.h4 {
   font-size: 1.1rem;
   font-weight: 200;
-  letter-spacing: .15px;
+  letter-spacing: 0.15px;
```

Comment:
Absolute size? Subsequent absolute sizes not noted in this file. Why use `letter-spacing` at all?